### DISCUSSION TITLE
Suggestion to Improve User Experience

### DISCUSSION THREAD

**Discussion by badr-issaad** [zyrouge/symphony#529] [Ideas]
I am sorry i can do that i did not see it i am an stupid, I will leave this msg here 🤣
 Hi,

I hope this message finds you well. I have a suggestion regarding the lower panel with buttons for "For You," "Songs," "Albums," "Artists," etc. Currently, I find it a bit cumbersome to access folders, which I prefer using. It takes extra steps to reach them as they are hidden. I would greatly appreciate it if there could be an option to edit the location of the buttons within the panel for easier accessibility.

Thank you for considering this enhancement to improve the user experience.

Best regards.![Screenshot_20240422_194423_Symphony.png](https://github.com/zyrouge/symphony/assets/79155030/ef52b035-11a0-46d5-8382-8a39190eb2e0)

**Comment by zyrouge** [comment]
> I am sorry i can do that i did not see it i am an stupid, I will leave this msg here 🤣

I think you have found the solution? I'll be closing this then.
